backend/src/libs/packages/encrypt/encrypt-service.ts

```diff
@@ -0,0 +1,25 @@
+import bcrypt from 'bcryptjs';
```

Comment:
```suggestion
import { genSalt, hash, compare } from 'bcryptjs';
```

Reply:
it is not allowed named import for compare function in this bcryptjs CommonJs module

Reply:
I've done desructurization
const { compare: comparePass, genSalt, hash } = bcrypt;